midealocal/device.py

```diff
@@ -231,7 +231,7 @@ def refresh_status(self, wait_response: bool = False) -> None:
 
     def pre_process_message(self, msg: bytes) -> bool:
         if msg[9] == MessageType.query_appliance:
-            message = MessageApplianceResponse(msg)
+            message = MessageApplianceResponse(bytearray(msg))

```

Comment:
I'm not sure if this is needed.
Especially because hen we will type the library this will create an error.

Reply:
MessageApplianceResponse expects bytearray and msg is bytes. That was due to type error

Reply:
Please git merge with latest code, `pre_process_message` is now `bytearray` ;-)